fe/estimator_abfe.py

```diff
@@ -153,13 +160,16 @@ def simulate(lamb, box, x0, v0, final_potentials, integrator, barostat, equil_st
     for obs in du_dp_obs:
         grads.append(obs.avg_du_dp())
 
-    result = SimulationResult(
-        xs=xs,
-        boxes=boxes,
-        du_dps=grads,
-        lambda_us=full_us,
-    )
-
+    with NamedTemporaryFile(suffix=".xtc") as temp:
+        traj = mdtraj.Trajectory(xs, mdtraj.Topology.from_openmm(openmm_topo))
```

Comment:
You can set topology=None here:

```
mdtraj.Trajectory(xs, topology=None)
```

ex:

``` python
xyz=np.random.rand(100,50,3)
boxes = np.random.rand(100,3,3)
traj = mdtraj.Trajectory(xyz=xyz, topology=None)
traj.unitcell_vectors=boxes
>>> traj.save_xtc('debug.xtc')
```

xtc format does not store anything about the topology.